libs/MeshKernelApi/tests/src/ApiTest.cpp

```diff
@@ -204,30 +204,16 @@ TEST_F(ApiTests, FlipEdges_WithALandBoundary_ShouldFlipEdges)
     const int projectToLandBoundaryOption = 1;
     meshkernelapi::GeometryList selectingPolygon{};
 
-    std::unique_ptr<double> const xCoordinates(new double[4]{
-        -0.5,
-        -0.5,
-        4.0,
-        meshkernel::constants::missing::doubleValue});
-
-    std::unique_ptr<double> const yCoordinates(new double[4]{
-        3.0,
-        -0.5,
-        -0.5,
-        meshkernel::constants::missing::doubleValue});
-
-    std::unique_ptr<double> const zCoordinates(new double[4]{
-        0.0,
-        0.0,
-        0.0,
-        meshkernel::constants::missing::doubleValue});
+    std::vector xCoordinates{-0.5, -0.5, 4.0, meshkernel::constants::missing::doubleValue};
```

Comment:
Shall we not use template arg deduction here and be explicit about the typename?